port/aix/omrvmem.c

```diff
@@ -949,11 +949,13 @@ omrvmem_find_valid_page_size(struct OMRPortLibrary *portLibrary, uintptr_t mode,
 
 	if (0 != validPageSize) {
 		/* For executable pages search through the list of supported page sizes only if
+		 * - request is for 64K pages, or
 		 * - request is for 16M pages, and
 		 * - 64 bit system OR (32 bit system AND CodeCacheConsolidation is enabled)
 		 */
 #if defined(OMR_ENV_DATA64)
 		if ((OMRPORT_VMEM_MEMORY_MODE_EXECUTE != (OMRPORT_VMEM_MEMORY_MODE_EXECUTE & mode)) ||
+			(SIXTY_FOUR_K == validPageSize) ||
 			(SIXTEEN_M == validPageSize))
```

Comment:
edited code should be reformatted as per coding standards: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions
```suggestion
		if ((OMRPORT_VMEM_MEMORY_MODE_EXECUTE != (OMRPORT_VMEM_MEMORY_MODE_EXECUTE & mode))
			|| (SIXTY_FOUR_K == validPageSize)
			|| (SIXTEEN_M == validPageSize)
```